README.md

```diff
@@ -1,2 +1,31 @@
 # Portfolio-project-set-up-and-mobile-version-skeleton
 Project: To parse 2 sections from a Figma portfolio design.
+
+# Project Name
+
+- Setup and mobile version skeleton
```

Comment:
- [ ] Please remove that because you already have the project name and description above.